src/hooks/v1/Projects.ts

```diff
@@ -28,26 +28,25 @@ import useSubgraphQuery, { useInfiniteSubgraphQuery } from '../SubgraphQuery'
 
 interface ProjectsOptions {
   pageNumber?: number
-  projectId?: BigNumber
-  handle?: string
-  uri?: string
+  projectId?: number
   orderBy?: 'createdAt' | 'currentBalance' | 'totalPaid'
   orderDirection?: 'asc' | 'desc'
   pageSize?: number
   state?: ProjectState
   keys?: (keyof Project)[]
   terminalVersion?: V1TerminalVersion
-  searchText?: string
+  cv?: Project['cv']
```

Comment:
**suggestion** thoughts on `contractVersion`?

Reply:
@aeolianeth generally i prefer more verbose names too. though since this property will probably get used pretty ubiquitously, i'm leaning towards brevity